core/src/kura.rs

```diff
@@ -18,22 +18,18 @@ use iroha_data_model::block::VersionedCommittedBlock;
 use iroha_logger::prelude::*;
 use iroha_version::scale::{DecodeVersioned, EncodeVersioned};
 use parking_lot::Mutex;
-use serde::{de::DeserializeSeed, Serialize};
 
-use crate::{
-    handler::ThreadHandler,
-    wsv::{WorldStateView, WsvSeed},
-};
+use crate::handler::ThreadHandler;
 
 const INDEX_FILE_NAME: &str = "blocks.index";
 const DATA_FILE_NAME: &str = "blocks.data";
-const SNAPSHOT_FILE_NAME: &str = "snapshot.data";
 const LOCK_FILE_NAME: &str = "kura.lock";
 
 /// The interface of Kura subsystem
 #[derive(Debug)]
 pub struct Kura {
     /// The mode of initialisation of [`Kura`].
+    #[allow(dead_code)]
     mode: Mode,
```

Comment:
Why not remove it?

```suggestion
```

Reply:
We still have plans to bring it back. I’m going to work on it after vacations.